src/quo/design_system/colors.cljs

```diff
@@ -62,7 +62,7 @@
    :interactive-02 "rgba(35,37,47,1)"
    :interactive-03 "rgba(255,255,255,0.1)"
    :interactive-04 "rgba(131,140,145,1)"
-   :ui-background  "rgba(20,20,20,1)"
+   :ui-background  "rgba(13,22,37,1)"
```

Comment:
Hi, why changing `quo/design_system/colors`? (this will also affect old ui)
And we should only use `quo2.colors` for new ui.

Reply:
For [this ](https://discord.com/channels/624205794384281629/626744384565674034/1016761652080824453)request

Reply:
> For this request
> @Mobile any chance we can get the default background of the testing area to be neutral 90?

We should not change color's value, we should change which color to use for preview background

Reply:
Correct, Thanks Parvesh!
I'll do it now

Reply:
Done

Reply:
> Done

[src/quo/design_system/colors.cljs](https://github.com/status-im/status-mobile/pull/13700/files#diff-034ed1cf36e210d1c972dcd619946509eb41afefec8b8d069d0dadbc84d60aba) still has changes, please revert it.